src/components/controls/types.ts

```diff
@@ -0,0 +1,16 @@
+/* Public types of controls */
+
+export type TextInputView = 'normal' | 'clear';
+
+export type TextInputSize = 's' | 'm' | 'l' | 'xl';
+
+export type TextInputPin =
```

Comment:
We also should have aliases for TextArea: `export type TextAreaView = TextInputView`